spec/server.test.js

```diff
@@ -0,0 +1,20 @@
+const request = require('supertest');
+const server = require('../server/app.js');
```

Comment:
maybe rename this variable to app so that it isn't confused with server/server.js